config/locales/forms/en.yml

```diff
@@ -13,7 +13,7 @@ en:
       caution_delete: If you delete your backup codes you will no longer be able to
         use them to sign in.
       confirm_delete: Are you sure you want to delete your backup codes?
-      download: Download
+      download: Download (text file)
```

Comment:
This is also used on the page to download MFA backup codes . Are we intending for the text to change there as well, or should we create a separate text for the personal key page ?

Reply:
Moved out to a new strings file entry and reverted this string to the original.